Quorum/check_proposal.py

```diff
@@ -45,7 +47,7 @@ def load_config(config_path: str) -> dict[str, Any] | None:
         pp.pretty_print(f"Failed to parse given config file {config_path}:\n{e}", pp.Colors.FAILURE)
 
 
-def proposals_check(customer: str, chain_name: str, proposal_addresses: list[str]) -> None:
+def proposals_check(customer: str, chain_name: str, proposal_addresses: list[str], providers: list[PriceFeedProviderBase]) -> None:
```

Comment:
Again, for now, it doesn't seem like that big of a deal but I don't like that we need to pass providers just for a single check out of all other checks. I think letting the PriceFeedCheck itself have access to the config would have been better. I can live with this for now but as we'll implement more and more checks, extending the parameters to this function might become really ugly.

Reply:
Agree but its much better for testing to make it input instead of a dependancy

Reply:
So i go with that solution